lib/sqlalchemy/engine/cursor.py

```diff
@@ -919,8 +906,8 @@ def __getstate__(self):
     def __setstate__(self, state):
         self._processors = [None for _ in range(len(state["_keys"]))]
         self._keymap = state["_keymap"]
-
         self._keymap_by_result_column_idx = None
+        self._key_to_index = self._make_key_to_index(self._keymap, MD_INDEX)
```

Comment:
**mike bayer** (zzzeek) wrote:

good this maintains pickle compat

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4575